packages/dds/sequence/src/test/intervalCollection.spec.ts

```diff
@@ -186,6 +175,61 @@ describe("SharedString interval collections", () => {
 			]);
 		});
 
+		describe("changing endpoints and/or properties", () => {
```

Comment:
Should there be tests added to intervalCollection.events.spec.ts? @Abe27342

Reply:
I figured I would add more tests for events when I update the events to match the new usage, but for now I was planning to add at least one test to verify that changing both endpoints and properties emits both events.